osu.Framework/Graphics/Containers/CompositeDrawable.cs

```diff
@@ -1339,6 +1340,38 @@ protected set
             }
         }
 
+        private float cornerExponent = 2.5f;
+
+        /// <summary>
+        /// Determines how gentle the curve of the corner straightens. A value of 2 results in
+        /// circular arcs, a value of 2.5 (default) results in something closer to apple's "continuous corner".
+        /// Values between 2 and 10 result in varying degrees of "continuousness", where larger values are smoother.
+        /// Values between 1 and 2 result in a "flatter" appearance than round corners.
+        /// Values between 0 and 1 result in a concave, round corner as opposed to a convex round corner,
+        /// where a value of 0.5 is a circular concave arc.
+        /// Only has an effect when <see cref="Masking"/> is true and <see cref="CornerRadius"/> is non-zero.
+        /// </summary>
+        public float CornerExponent
+        {
+            get => cornerExponent;
+            protected set
+            {
+                if (!Precision.DefinitelyBigger(value, 0) || value > 10) {
```

Comment:
Comparisons against 0 should be accurate.

```suggestion
                if (value <= 0 || value > 10) {
```

Reply:
I don't want to allow values arbitrarily close to 0, though, because numerics become unstable.